ruby/cross_service_examples/item_tracker/README.md

```diff
@@ -0,0 +1,102 @@
+|POST                | /api/items:report    |Create report | RDSDataService     |execute_statement(*SQL*)|
+
+## 2. Frontend
+The frontend code is written in JavaScript and implemented using the React framework.
+For more information, see the [React Elwing Client README](../../../resources/clients/react/elwing/README.md).
+
+## 3. Database
+This example relies on a MySQL 5.7 database implemented using Amazon Aurora, a serverless relational database management system (RDBMS).
+The Aurora cluster is deployed using the AWS Cloud Development Kit (AWS CDK).
+For more information, see the [Aurora Serverless App CDK script README.md](../../../resources/cdk/aurora_serverless_app/README.md).
+
+## Prerequisites
+To explore this example, you must first:
+1. Create account and configure credentials using [instructions in the top-level README.md](../../../README.md#invoke-example-code).
+2. Install Ruby and resolve dependencies using [instructions in the Ruby README.md](../../../ruby/README.md).
+
+## Create the resources
+Set up the AWS resources needed to run this example as follows:
+1. Create AWS infrastructure using [instructions in the Aurora Serverless App README.md](../../../resources/cdk/aurora_serverless_app/README.md).
+   * NOTE: Store the following output values in the `env/config.yml` file: `CLUSTER_ARN`, `SECRET_ARN`, and `DATABASE`.
+2. Populate your database with a table and data using the following commands:
+     ```bash
+     cd ruby/cross_service_examples/item-tracker/helpers
+     ruby create_table.rb # checks for database and creates a new table
+     ruby populate_data.rb # creates a table with 10 records
+     ```
+3. Register your email address with Amazon Simple Email Service (SES) using the instructions in [Creating and verifying identities in Amazon SES](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html)
+   * NOTE: Use this email for the `sender_email` and `recipient_email` fields in your `env/config.yml` file.
+
+## Build the code
+To view this example in its entirety, you must:
+1. Start the frontend React application using the following commands:
+    ```bash
+     cd resources/clients/elwing # located within Resources sub-directory
+     npm install # first time only
+     npm start # launches a browser session on localhost:3000 
+    ```
+2. Open a second tab in your terminal and run the following command:
+    ```bash
+    cd ruby/cross_service_examples/item-tracker
+    ruby app.rb # starts REST API listening on port 8080
+    ```
+Now, visit http://localhost:3000/item_tracker to view your working application.
+
+## Test the code
+Test this application manually via the frontend or automatically via the RSpec tests.
+
+### Manual validation (via frontend React app)
+Once the app is started, manually validate the following behaviors on the frontend:
+* Filter items by All, Archived, and Active (not archived).
+* Archive an item by clicking the Archive button.
```

Comment:
clicking --> choosing

Reply:
Not a huge fan of this one but I comply :)